src/components/Authentication/OAuth.vue

```diff
@@ -0,0 +1,73 @@
+<template lang="pug">
+.oauth-widget(v-if="providers.length")
```

Comment:
We don't use the `oauth-widget` class anywhere, so it can be removed. Also, we should move this `v-if` up to the parent components, I think it may generally be a code smell to have the top level element contain `v-if`. So, we can get rid of this element altogether.

Reply:
I prefer these over plain divs because it makes navigating the inspector easier.  You can see by class exactly where components start.  This is especially handy to me because vuetify components seems to add 3-5 layers of wrapping, so the tree is kind of noisy.

Vuetify also follows this practice, wrapping each component in a div with class `v-component-name`

Reply:
Fair enough, we can just move the class down to the `v-container`.